pkg/common/common.go

```diff
@@ -78,4 +80,27 @@ const (
 
 	// DefaultResyncPeriod sets a 10 minute resync period, used in the controller pkg and the controller cmd executable
 	DefaultResyncPeriod = 10 * time.Minute
+
+	//Importer enum for ControllerType
+	Importer ControllerType = 0
+	//Cloner enum for ControllerType
+	Cloner ControllerType = 1
+	//Uploader enum for ControllerType
+	Uploader ControllerType = 2
+	//Unknown enum for ControllerType
+	Unknown ControllerType = 3
 )
+
+func (controller ControllerType) getControllerName() string {
```

Comment:
If we just modify the Controller to have a member that points to this func in the main controller class it might be even easier?  Or just use a string to begin with, the only downside to that is when I modified all the logging it does get a little noisy.